lms/djangoapps/courseware/tests/test_course_info.py

```diff
@@ -85,6 +94,52 @@ def test_nonexistent_course(self):
         self.assertEqual(response.status_code, 404)
 
 
+class CourseInfoTestCaseCCX(SharedModuleStoreTestCase, LoginEnrollmentTestCase):
+    """
+    Test for unenrolled student tries to access ccx.
+    Note: Only CCX coach can enroll a student in CCX. In sum self-registration not allowed.
+    """
+
+    MODULESTORE = TEST_DATA_SPLIT_MODULESTORE
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Set up course for tests
```

Comment:
I don't think pylint requires docstrings on test methods. If pylint *does* require docstrings, we should change that, because these docstrings are not very useful.